FinalProject/StudentTests.java

```diff
@@ -489,6 +489,32 @@ void ratingCountPerProfTest2() {
     assertEquals(v1, v2);
   }
 
+  // round the average rating to two decimal number
+  @Test
+  @DisplayName("RatingCountPerProf Test 3")
+  void ratingCountPerProfTest3() {
+    Parser parser = new Parser("");
```

Comment:
Can you add the fields here for the parser? Some people might query the fields hashmap
add at line 495:
```java
    HashMap<String, Integer> fields = new HashMap<>();
    fields.put("professor_name", 0);
    fields.put("school_name", 1);
    fields.put("department_name", 2);
    fields.put("post_date", 3);
    fields.put("student_star", 4);
    fields.put("student_difficult", 5);
    fields.put("comments", 6);
    fields.put("gender", 7);

    parser.fields = fields;
```